targetclid.8

```diff
@@ -0,0 +1,87 @@
+.TH targetclid 8
+.SH NAME
+.B targetclid
+\- daemon component for targetcli
+.SH DESCRIPTION
+.B targetclid
+is the daemon component of targetcli, which will help retain state of various
+configfs object in memory, hence any new request/command can directly use the
+in memory objects instead of reconstructing them by parsing through the entire
+configfs files again and again for each and every single command. This will
+greatly improve the overall execution time taken by targetcli commands at scale.
+
+.SH USAGE
+.B targetclid [cmd]
+.br
+.B "--help"
+for additional usage information.
+.br
+.B "--version"
+for version information.
+.SH QUICKSTART & EXAMPLES
+.TP
+To start using the daemon, one need to enable targetclid socket,
+.br
+$ systemctl enable targetclid.socket
+.TP
+If you would like to use the daemonized approach as default method then,
+.br
+$ targetcli set global auto_use_daemon=true
```

Comment:
Not sure should we also mention that there has one daemon mode stuff in the targetcli.8 ?